models/src/main/java/com/vimeo/networking2/Folder.kt

```diff
@@ -99,3 +99,8 @@ val Folder.slackLanguagePreferenceType: SlackLanguagePreferenceType
  */
 val Folder.slackUserPreferenceType: SlackUserPreferenceType
     get() = slackUserPreference.asEnum(SlackUserPreferenceType.UNKNOWN)
+
+/**
+ * Get the teams_permission uri by using the folder's own uri.
+ */
+val Folder.teamPermissionsUri: String? get() = uri?.let { "$it/team_permissions" }
```

Comment:
I'm not sure this extension is necessary since the URI should be obtained via `folder?.metadata?.connections?.teamPermissions?.uri`

Reply:
I opted for this because of the annoying field filtering behavior where the connection isn't returned unless it's included in a filter.